### ISSUE TITLE
A11y_PowershellEditorServices_Terminal_ScreenReader: NVDAdoes not read the expanded/collapsed status of the Combo box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#61700]
**Environment Details:**
VSCode Version : 1.25.1
OS Version : Win10

**Additional Details:**
Screenreader MAS1.3.1

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate "PowerShell Integrated Console" List item.
4.Start NVDA & observe whether narrator reads expanded/collapsed status of the Combo box or not.

**Actual:**
NVDA does not read the expanded/collapsed status of the Combo box (i.e."PowerShell Integrated Console")

**Expected:**
NVDA should read the expanded/collapsed status of the Combo box (i.e."PowerShell Integrated Console")

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not able to know the state of the combo box.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Does this issue occur when all extensions are disabled?: Yes

[A11y_PowershellEditorServices_Terminal_ScreenReader.pptx](https://github.com/Microsoft/vscode/files/2510128/A11y_PowershellEditorServices_Terminal_ScreenReader.pptx)

**Comment by cleidigh** [comment]
@Tyriar 
This is the ongoing battle with Electron/Chromium/ARIA quirks but some of which we can work around in 2.x versions. But I will look at it.

**Comment by cleidigh** [comment]
@AccessibilityTestingTeam-TCS 
I just tested with Windows 10 and NVDA 2018.2.1
The reader reads "collapsed" for the terminal as well as other drop downs.

![image](https://user-images.githubusercontent.com/25272315/47525709-ef23c800-d86b-11e8-8e3a-9daf8bf8d4cb.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
In the latest Version: 1.28.2 issue does not repro hence closing this bug.
![2726852_verified](https://user-images.githubusercontent.com/39204909/47565759-303cda80-d947-11e8-9c97-7a27d426c44e.png)
